libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -14,7 +14,9 @@
 #define INS_MAX_INSTANCES 3
 #define INS_MAX_BACKENDS  6
 #define INS_MAX_NOTCHES 4
-#define INS_VIBRATION_CHECK_INSTANCES 2
+#ifndef INS_VIBRATION_CHECK_INSTANCES
```

Comment:
use HAL_MEM_CLASS to enable all for >= HAL_MEM_CLASS_300, and just 1 otherwise

Reply:
Done.